demo/kitchen_sink/run.py

```diff
@@ -155,6 +154,7 @@ def fn(
     cache_examples=False,
     description="Try out all the components!",
     article="Learn more about [Gradio](http://gradio.app)",
```

Comment:
`cache_examples` is set to both True above and False here, causing an error

Reply:
Whoops thanks